ddl/syncer.go

```diff
@@ -116,7 +116,11 @@ func (s *schemaVersionSyncer) putKV(ctx goctx.Context, retryCnt int, key, val st
 
 // Init implements SchemaSyncer.Init interface.
 func (s *schemaVersionSyncer) Init(ctx goctx.Context) error {
-	_, err := s.etcdCli.Txn(ctx).
+	startTime := time.Now()
+	var err error
+	defer deploySyncerHistogram.WithLabelValues(syncerInit, retLabel(err)).Observe(time.Since(startTime).Seconds())
```

Comment:
`time.Since(startTime)` is evaluated before the deferred function run.

We have to wrap this line into a function.